### PR TITLE
use environment variable `PARL_BACKEND` to specify backend framework

### LINKED ISSUE #530: [backend] Support the environment variable for backend
PARL supports different deep learning frameworks now. Sometimes people have installed several frameworks but they want to use the specified framework.
To meet this requirement, we refer to what Keras did at supporting multiple frameworks. 
https://stackoverflow.com/questions/42177658/how-to-switch-backend-with-keras-from-tensorflow-to-theano

### PR DESCRIPTION
**Pull request by rical730**
fix https://github.com/PaddlePaddle/PARL/issues/530

### CODE REVIEW

File: parl/utils/utils.py [hunk 1]
```diff
@@ -134,10 +135,22 @@ def isnotebook():
...
           "Please make sure the version is good with your code."
     import paddle
     from paddle import fluid
 
     paddle_version = get_fluid_version()
     assert paddle_version >= 200 or paddle_version == 0, err
+
+
+def check_version_for_fluid():
+    """check paddle version if the code want to use fluid parl
```
Comment (TomorrowIsAnOtherDay) [inline parl/utils/utils.py]: check the paddle version if the code requires fluid-based implementation.

File: parl/utils/utils.py [hunk 2]
```diff
@@ -134,10 +135,22 @@ def isnotebook():
...
+    """
+    err = "To use fluid version of examples, PARL requires paddle version < 2.0.0. " \
+          "Please make sure the version is good with your code."
```
Comment (TomorrowIsAnOtherDay) [inline parl/utils/utils.py]: Please make sure the version is `consistent` with your code.

File: parl/utils/utils.py (hunk 1 above)
Comment (rical730) [reply parl/utils/utils.py]: done

File: parl/utils/utils.py (hunk 2 above)
Comment (rical730) [reply parl/utils/utils.py]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Pull request merged**